.travis.yml

```diff
@@ -18,4 +18,4 @@ install:
 
 script:
   - yarn lint:js
-  - yarn test
+  - yarn test --forceExit
```

Comment:
@rwjblue the startApp sometimes leaves open handles.  I've done several hours of investigation.  I'm spent.  😪